app/src/test/java/com/github/se/wanderpals/repository/TripsRepositoryTest.kt

```diff
@@ -68,7 +136,7 @@ class TripsRepositoryTest {
 
         withTimeout(10000) {
           assertFalse(repository.addTripId(nonExistentTripId))
-          assertFalse(repository.removeTripId(nonExistentTripId))
+          assertTrue(repository.removeTripId(nonExistentTripId))
```

Comment:
Why did this change to return true now ?

Reply:
The previous version of removeTripid, failed if it couldn't find the TripId,which we wished to delete, however this is not the case in mutliple other methods, which for consistency will return true, if the target of deletion doesn't exist. So I just fixed this method to follow the others. Hope this clears things out)

Reply:
Thx for the very clear explanation